### PR TITLE
[Feature][Maxell] site animations

### PR DESCRIPTION
**Pull request by maxellmilay**
What's in the PR?
 - added animations in the home and about page

Demo Link:
https://www.loom.com/share/f05835e3f7c04e77ad6c87a037d50445

### CODE REVIEW

File: styles/about.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.history-header {
```
Comment (ezerssss) [inline styles/about.css]: change guro ang naming nga mag involve ug animation? `history-header-animation`

File: styles/index.css [hunk 2]
```diff
@@ -32,3 +32,18 @@
 .handle {
   grid-area: handle;
 }
+
+.landing-section-text {
```
Comment (ezerssss) [inline styles/index.css]: `-animation`

**Review by ezerssss — APPROVED** [review]

File: styles/about.css (hunk 1 above)
Comment (maxellmilay) [reply styles/about.css]: resolved

File: styles/index.css (hunk 2 above)
Comment (maxellmilay) [reply styles/index.css]: resolved

**Pull request merged**